### ISSUE TITLE
Sync not working as expected

### ISSUE THREAD

**Issue by srodrig0209** [dirkgroenen/mopidy-mopify#80]
Maybe I'm not running mopidy correctly, which I hope is the case, but when attempting to use the sync service, the GUI didn't return an error except when I clicked `GET`.

I opened the Chrome console and saw a 500 error(would be nice to see an error popup when the server returns this), so I stopped mopidy and re-ran it as `mopidy -v` and found the culprit.

```
ERROR    2015-03-24 11:09:08,252 [79729:HttpServer] tornado.application
  Uncaught exception POST /mopify/sync/spotify?

[truncated]

Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/tornado/web.py", line 1332, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/usr/local/lib/python2.7/site-packages/mopidy_mopify/sync.py", line 53, in post
    spotify = Spotify()
  File "/usr/local/lib/python2.7/site-packages/mopidy_mopify/sync.py", line 82, in __init__
    self.syncini = ConfigObj(self.syncfile, encoding='UTF8', create_empty=True)
  File "/usr/local/lib/python2.7/site-packages/configobj.py", line 1229, in __init__
    self._load(infile, configspec)
  File "/usr/local/lib/python2.7/site-packages/configobj.py", line 1246, in _load
    with open(infile, 'w') as h:
IOError: [Errno 2] No such file or directory: u'/Users/USER/.config/mopidy-mopify/sync.ini'
```

Created the `mopidy-mopify` directory and then the `sync.ini` file and then all was well.

Simply adding a blurb to the wiki about creating the `mopidy-mopify` folder should be sufficient.

**Comment by dirkgroenen** [comment]
Ah looks like the file isn't created automatically (which it should). Which version are you running

**Comment by srodrig0209** [comment]
Well, I just installed it today, so I've got the latest.

**Comment by dirkgroenen** [comment]
Ok, I'll fix this as soon as possible.

**Comment by totokaka** [comment]
I can confirm the `~mopidy/.config/mopidy-mopify` folder is not created automatically, and that the sync service doesn't work before it's created.

Creating the folder made sync work, at least partially. I'll have a close look at it later.

**Comment by dirkgroenen** [comment]
Looks like the Sync's constructor isn't called. This constructor should create the proper directories and files. Again, I'll fix it as soon as possible, but everybody is of course welcome to fix it and create a pull request.

**Comment by dirkgroenen** [comment]
Fixed for next release ([1.4.0](https://github.com/dirkgroenen/mopidy-mopify/milestones/1.4.0))
